cirq/ops/common_channels.py

```diff
@@ -146,7 +144,7 @@ def __init__(self, p) -> None:
         self._p = p
         self._delegate = AsymmetricDepolarizingChannel(p / 3, p / 3, p / 3)
 
-    def _mixture_(self) -> Sequence[Tuple[float, np.ndarray]]:
+    def _mixture_(self) -> Sequence[Tuple[float, raw_types.Gate]]:
```

Comment:
We should re-order the tuple to be `[object, probability]` instead of `[probability, object]`, so it is compatible with `{A: p1, B: p2}.entries()`. You would never make a dictionary mapping probabilities to items (because duplicates), but vice versa is conceivable.